templates/blog/post-detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <a href="{% url 'blog:home' %}" class="home-button">Home</a>

```

Comment:
The URL tag uses 'blog:home', but it should be 'blog:index' to match the URL pattern defined in `urls.py`.